src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/UnlockableItemStack.java

```diff
@@ -0,0 +1,46 @@
+package io.github.thebusybiscuit.slimefun4.implementation.guide;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.ParametersAreNonnullByDefault;
+
+import org.bukkit.Material;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.meta.ItemMeta;
+
+import io.github.bakedlibs.dough.common.ChatColors;
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItem;
+import io.github.thebusybiscuit.slimefun4.api.items.SlimefunItemStack;
+import io.github.thebusybiscuit.slimefun4.core.services.localization.SlimefunLocalization;
+import io.github.thebusybiscuit.slimefun4.implementation.Slimefun;
+
+/**
+ * An extension of {@link SlimefunItemStack} that indicates the item is unlockable in the guide.
+ * This is only used by {@link SurvivalSlimefunGuide}.
+ *
+ * @author RobotHanzo
+ * 
+ * @see SurvivalSlimefunGuide
+ */
+class UnlockableItemStack extends SlimefunItemStack {
```

Comment:
```suggestion
public class UnlockableItemStack extends SlimefunItemStack {
```
Unless there is a specific reason this is package private I feel it should be made public. Other classes with similar usage are also public so it's a strange difference to make this one not so.